packages/react-components/react-table/stories/DataGrid/KeyboardColumnResizing.stories.tsx

```diff
@@ -25,8 +25,6 @@ import {
   MenuPopover,
   MenuTrigger,
   MenuItem,
-  useFocusFinders,
```

Comment:
as  part of this PR, let's unify all the resizing examples to controlled/uncontrolled, we shouldn't be showcasing any resizing examples without keyboard since it would not be accessible by default

Reply:
yea that makes sense, I'll look into it :)